src/components/v2/appDetails/AppDetails.component.tsx

```diff
@@ -39,7 +43,27 @@ const AppDetailsComponent = ({
             appDetails?.appType?.toString() != AppType.EXTERNAL_HELM_CHART.toString(),
         (event) => setStreamData(JSON.parse(event.data)),
     );
-
+    if(appDeleteError.length > 0){
```

Comment:
We can simply add a check of if(appDeleteError) no need to check length. In case of appDeleteError is undefined current implementation will throw error